creativecloud/features/firefly/firefly-interactive.js

```diff
@@ -0,0 +1,264 @@
+import { setLibs } from '../../scripts/utils.js';
+
+const miloLibs = setLibs('/libs');
+const { createTag } = await import(`${miloLibs}/utils/utils.js`);
+const { createSelectorTray, createEnticement, createPromptField } = await import('../interactive-elements/interactive-elements.js');
+
+function eventOnGenerate(generateButton, media) {
+  generateButton.addEventListener('click', async () => {
+    const textToImageButton = media.querySelector('#TextToImage');
+    const textEffectsButton = media.querySelector('#TextEffects');
+    let prompt = '';
+    let promptDefault = false;
+    const userprompt = media.querySelector('.promptText')?.value;
+    const placeholderprompt = media.querySelector('.promptText')?.getAttribute('placeholder');
+    if (userprompt === '') {
+      promptDefault = true;
+      prompt = placeholderprompt;
+    } else {
+      prompt = userprompt;
+    }
+    if (textToImageButton.classList.contains('selected')) {
+      if (promptDefault) {
+        generateButton.setAttribute('daa-ll', 'SubmitTextToImage');
+      } else {
+        generateButton.setAttribute('daa-ll', 'SubmitTextToImageUserContent');
+      }
+      const { signIn } = await import('./firefly-susi.js');
+      signIn(prompt, 'goToFirefly');
+    }
+    if (textEffectsButton.classList.contains('selected')) {
+      if (promptDefault) {
+        generateButton.setAttribute('daa-ll', 'SubmitTextEffects');
+      } else {
+        generateButton.setAttribute('daa-ll', 'SubmitTextEffectsUserContent');
+      }
+      // eslint-disable-next-line import/no-cycle
+      const { signIn } = await import('./firefly-susi.js');
+      signIn(prompt, 'goToFireflyEffects');
+    }
+  });
+}
+
+function createGenFillPrompt(element) {
+  const genfillPrompt = createTag('div', { class: 'genfillPrompt' });
```

Comment:
```suggestion
  const genfillPrompt = createTag('div', { class: 'genfillPrompt' });
  ```suggestion
  const promprConfig = element?.innerText?.split('|')[0].split('(');
```
```

Reply:
done